explorer/client/src/components/Table.tsx

```diff
@@ -11,55 +11,22 @@ import TableHead from '@material-ui/core/TableHead'
 import TableRow from '@material-ui/core/TableRow'
 import MuiTableCell from '@material-ui/core/TableCell'
 import TableCell, { Column } from './Table/TableCell'
-import TablePagination from '@material-ui/core/TablePagination'
+import TablePagination, {
+  TablePaginationProps,
+} from '@material-ui/core/TablePagination'
 import Paper from '@material-ui/core/Paper'
 import PaginationActions from './Table/PaginationActions'
 
 export const DEFAULT_ROWS_PER_PAGE = 10
 export const DEFAULT_CURRENT_PAGE = 1
 
-interface MsgProps {
-  colCount: number
-  msg?: string
-}
-
-const Loading: React.FC<MsgProps> = ({ colCount, msg }) => {
-  return (
-    <TableRow>
-      <MuiTableCell component="th" scope="row" colSpan={colCount}>
-        {msg ?? 'Loading...'}
-      </MuiTableCell>
-    </TableRow>
-  )
-}
-
-const Empty: React.FC<MsgProps> = ({ colCount, msg }) => {
-  return (
-    <TableRow>
-      <MuiTableCell component="th" scope="row" colSpan={colCount}>
-        {msg ?? 'There are no results added to the Explorer yet.'}
-      </MuiTableCell>
-    </TableRow>
-  )
-}
-
-const Error: React.FC<MsgProps> = ({ colCount, msg }) => {
-  return (
-    <TableRow>
-      <MuiTableCell component="th" scope="row" colSpan={colCount}>
-        {msg ?? 'Error loading resources.'}
-      </MuiTableCell>
-    </TableRow>
-  )
-}
-
 const styles = (theme: Theme) =>
   createStyles({
     header: {
       backgroundColor: theme.palette.grey['50'],
     },
     table: {
-      minHeight: 150,
+      minHeight: 550,
```

Comment:
550px breaks the layout if we have less list items in a table.

<img width="1792" alt="Screenshot 2020-04-27 at 18 03 48" src="https://user-images.githubusercontent.com/32734780/80395607-c131a080-88b3-11ea-94ef-6b6b4d0ee01a.png">

Reply:
Updated the implementation so that it uses the original min height + last known table height

![Screen Shot 2020-04-30 at 4 17 21 PM](https://user-images.githubusercontent.com/680789/80768041-2d9fe000-8afe-11ea-9ef1-bf5646ec2b0a.png)